compiler-plugin/src/main/java/io/ballerina/stdlib/websub/task/ListenerInitAnalysisTask.java

```diff
@@ -107,17 +108,24 @@ private Optional<ListenerDeclarationNode> getImplicitListenerNode(ImplicitNewExp
         return Optional.empty();
     }
 
-    private void verifyListenerArgType(SyntaxNodeAnalysisContext context, NodeLocation location,
+    private void verifyListenerArgType(SyntaxNodeAnalysisContext context,
                                        SeparatedNodeList<FunctionArgumentNode> functionArgs) {
         // two args are valid only if the first arg is numeric (i.e, port and config)
         if (functionArgs.size() > 1) {
             PositionalArgumentNode firstArg = (PositionalArgumentNode) functionArgs.get(0);
```

Comment:
Here, are we safe to cast this to a `PositionalArgumentNode` without checking it first?
```suggestion
            PositionalArgumentNode firstArg = (PositionalArgumentNode) functionArgs.get(0);
```

Reply:
Fixed with [e07e0e74597c03cfc839e32cb1c0eb182ff1fd1b](https://github.com/ballerina-platform/module-ballerina-websub/pull/652/commits/e07e0e74597c03cfc839e32cb1c0eb182ff1fd1b)